.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+on:
+  pull_request: ~
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Test (PHP ${{ matrix.php }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php: 7.4
+#          - php: 8.1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Docker compose
+        run: docker-compose up -d && tests/wait_for_crossbar.sh
```

If you switch to the new Compose, you can do
```suggestion
        run: docker compose up -d && docker compose up --wait crossbar
```